crates/store/re_types/definitions/rerun/datatypes/tensor_data.fbs

```diff
@@ -19,8 +19,15 @@ table TensorData (
   "attr.python.array_aliases": "npt.ArrayLike",
   "attr.rust.derive": "PartialEq,"
 ) {
-  /// The shape of the tensor, including optional names for each dimension.
-  shape: [rerun.datatypes.TensorDimension] (order: 200);
+  /// The shape of the tensor, i.e. the length of each dimension.
+  shape: [uint64] (order: 200);
+
+  /// The names of the dimensions of the tensor (optional).
+  ///
+  /// If set, should be the same length as [datatypes.TensorData.shape].
+  ///
+  /// Example: `["height", "width", "channel", "batch"]`.
+  names: [string] (order: 250, nullable);
```

Comment:
Or maybe we should call this `dim_names` to be explicit?

Reply:
Or maybe I regret ever starting this work and will never ever touch Python again in my life

Reply:
I think I prefer just `names`, I can't think of any other thing that is related to names in tensors, but I might be missing something?